bpaotu/bpaotu/templates/bpaotu/search.html

```diff
@@ -3,7 +3,51 @@
 {% load static from staticfiles %}
 
 {% block extra_head %}
+<script>
+// $(document).ready(function() {
+    var authenticate_users = function() {
+        // !!! NOTE: Do not hardcode the endpoints - so change it eventually.
+        // var bpa_endpoint = window.otu_search_config['ckan_base_url'] + '/user/me';
+        // var bpaotu_endpoint = window.otu_search_config['ckan_base_url'] + '/bpa/otu/otu_verification_endpoint';
+        var bpa_endpoint = 'https://localhost:8443/user/check_permissions';
+        var bpaotu_endpoint = 'https://localhost:8443/bpa/otu/otu_verification_endpoint';
+
+        $.ajax({
+            url: bpa_endpoint,
+            success: function(result) {
+                $.ajax({
+                    url: bpaotu_endpoint,
+                    data: { 'result': result },
+                    success: function(permission) {
+                        if (permission == 'Valid') {
+                        } else {
+                            window.document.write("Please log in to the Bioplatforms data portal \
+                                                    and ensure that you have access to the Ausmicro data to use the search tool.");
+                            window.stop();
+                        }
```

Comment:
don't use document.write, using jQuery and a target alert div (I think we already have one for other status messages)

Reply:
Fixed.